binary_transparency/firmware/cmd/flash_tool/flash_tool.go

```diff
@@ -117,6 +123,19 @@ func readUpdateFileFromFlags() (api.UpdatePackage, error) {
 	return up, nil
 }
 
+func checkSignature(up api.UpdatePackage) error {
+	stmt := api.FirmwareStatement{}
+	if err := json.NewDecoder(bytes.NewReader(up.ProofBundle.ManifestStatement)).Decode(&stmt); err != nil {
+		glog.Exitf("Failed to decode firmware statement: %q", err)
+	}
+
+	// "Verify" the signature:
```

Comment:
Can remove the quotes around verify (it makes it sounds like it's a bit fishy with them in :D)

Reply:
ACK